src/Widgets/DataLabelWidget/DataLabelWidgetComponent.tsx

```diff
@@ -73,3 +75,12 @@ function formatCurrency(value: unknown, currency: string | null) {
 
   return formatter.format(number)
 }
+
+function formatDatetime(value: unknown) {
+  if (value === null) return 'N/A'
+
+  const date = new Date(value as string)
```

Comment:
* [ ] Can editors break this by just playing around with settings in the UI or is value guaranteed to be a string? If we don't know for sure we should do a type check instead of casting.

Reply:
To my understanding `new Date(value)` can take any kind of `value` and return an invalid date (that is the check on line 83 `if (Number.isNaN(date.getTime())) return 'N/A'`). But I'll make this code more explicit to no longer need the cast.